rulesets/owasp_ruleset_functions.go

```diff
@@ -0,0 +1,893 @@
+package rulesets
+
+import (
+	"regexp"
+
+	"github.com/daveshanley/vacuum/model"
+	"github.com/daveshanley/vacuum/parser"
+)
+
+// rules taken from https://github.com/stoplightio/spectral-owasp-ruleset/blob/main/src/ruleset.ts
+
+func GetOWASPNoNumericIDsRule() *model.Rule {
+
+	// create a schema to match against.
+	opts := make(map[string]interface{})
+	// TODO: not exactly equal to the one in spectral
```

Comment:
Does this matter? How different is it?

Reply:
Spectral is cool, but vacuum is way cooler.. so I am fine if it hits the requirement of the OWASP rule but does not act identically to what came before in spectral.

Reply:
done